### ISSUE TITLE
cumination site cambro.tv broken

### ISSUE THREAD

**Issue by hehewtf** [dobbelina/repository.dobbelina#871]
videos not playing

loads far enough to trigger resolution/hertz change then hangs at 00:00 and nothing happens 

when stopping playback it freezes kodi having to kill the app process

**Comment by DJMix-I-Can** [comment]
Cambro is working fine on my Android device. Tried numerous videos....
You'll need to provide the following...

### OS environment
What OS, Device, version etc

1. Install Kodi Logfile Uploader addon from Offcial Kodi Repository under Program add-ons.
2. Enable Debug logging, (Settings - System - Logging - Enable Debug Logging)
3. Restart Kodi (Important)
4. Go into Cumination and open the site (wait for the error message)
5. Come out of cumination
6. Disable Debug logging, (Settings - System - Logging - Enable Debug Logging off)
7. launch Kodi Logfile uploader and upload your log. This will give you a url
8. provide the url from step 7 here

Any bugreports without logfile will be deleted.

**Comment by hehewtf** [comment]
if no one else has issues it might just be my modded kodi, I'll provide info regardless

nvidia shield pro 2019 model running android tv 11

using latest maven's kodinerds matrix build https://repo.kodinerds.net/index.php?action=list&scope=cat&item=Binary%20(arm64-v8a) 

maven build version is probably irrelevant though since cambro has never worked, multiple builds over a long timespan

might try an official kodi build just to test it later when I have time to set it up

log: https://paste.kodi.tv/zagaripesu.kodi

disregard my buffering settings in advancedsettings.xml
same outcome with default settings without an xml

basically just loading up a video, doesn't matter which one, tried like 10 different ones with the exact same results
waiting for it to start and giving up after screensaver starts, when I quit it freezes (at the end of log)

**Comment by DJMix-I-Can** [comment]
I just tested it with the latest Kodi Nerds Matrix version. It's a Vanilla installation, I added Dobbelinas repo address under file manager. Installed the repo & then installed Cumination. Once installed, I went into Cumination & CambroTV. Clicked on a video & it played without a problem. I tried numerous videos and it didn't give me any problems. I then installed Arctic Zephyr Mod 1.1.72 to see if that was causing you problems with the Embuary Helper. Nothing, the videos played fine without any tweaking on my part. I'm also using the Nvidia Shield 2019. Any chance you're using a VPN or an anti-ad blocker on the shield?

**Comment by hehewtf** [comment]
> I just tested it with the latest Kodi Nerds Matrix version. It's a Vanilla installation, I added Dobbelinas repo address under file manager. Installed the repo & then installed Cumination. Once installed, I went into Cumination & CambroTV. Clicked on a video & it played without a problem. I tried numerous videos and it didn't give me any problems. I then installed Arctic Zephyr Mod 1.1.72 to see if that was causing you problems with the Embuary Helper. Nothing, the videos played fine without any tweaking on my part. I'm also using the Nvidia Shield 2019. Any chance you're using a VPN or an anti-ad blocker on the shield?

thanks for the extra efforts of troubleshooting

no VPN and no form of adblocking
my Shield isn't rooted or anything, just a standard OTA build 

tested changing DNS in case that'd have any impact 
same deal on both Google DNS and cloudflare DNS with system restarts to hopefully having cleared DNS cache


no idea what the hell is going on since it's only happening with cambro and I have the same thing happen with a clean kodi install from google play 🤷‍♂️

videos work in web browsers on both my Shield and my phone so I doubt it's a connection issue

**Comment by DJMix-I-Can** [comment]
I didn't install my Kodi via Google Play, I downloaded the APK from [here](https://mirrors.kodi.tv/releases/android/arm64-v8a/kodi-19.4-Matrix-arm64-v8a.apk?https=1)
My Shield is also not rooted but I rolled it back to 8.2.3 after installing Experience 9.01 (Android 11). I also have my devices with static IP addresses & Google DNS. I also set my routers with the Google DNS. This is my Advanced Settings file if you want to try it out. I'm more curious about the disablehttp2 part of it. I had a similar issue awhile back and Gujal00 suggested I add that to my AdvancedSettings file. It started working after that... Maybe someone else could chime in. 
![AdvancedSettings](https://user-images.githubusercontent.com/35236592/197365294-147d479c-a669-4a89-81da-a01e379535e1.jpg)
